apps/src/sharedFunctionalBlocks.js

```diff
@@ -320,19 +318,141 @@ function installString(blockly, generator) {
  * condition-value pairs before the default value.
  */
 function installCond(blockly, generator, numPairs) {
```

Comment:
I should actually probably just get rid of it, since I think I'm only installing with 1.